### PR TITLE
Add --force_inject_library to command-line

### PR DESCRIPTION
**Pull request by dsanders11**
See discussion in #1138.

### CODE REVIEW

File: src/com/google/javascript/jscomp/CommandLineRunner.java [hunk 1]
```diff
@@ -690,6 +690,12 @@
         usage = "Allow injecting runtime libraries.")
     private boolean injectLibraries = true;
 
+    @Option(name = "--force_inject_library",
```
Comment (brad4d) [inline src/com/google/javascript/jscomp/CommandLineRunner.java]: Let's make this `--for

Comment (dsanders11) [inline src/com/google/javascript/jscomp/CommandLineRunner.java]: @brad4d, I think your sentence got cut off?

Comment (brad4d) [inline src/com/google/javascript/jscomp/CommandLineRunner.java]: Actually, I meant to cancel the comment.

**Comment by brad4d** [comment]
Testing looks good so far.
Taking it to the next level.

**Comment by brad4d** [comment]
Sent for internal review.

**Pull request closed without merging**